<?xml version="1.0"?>
<launch>


<node name="camera_0" pkg="cv_camera" type="cv_camera_node" respawn="true" respawn_delay="2" output="screen">
<param name="device_path" value="/dev/video0"/>
<param name="image_width" value="1280"/>
<param name="image_height" value="720"/>
<param name="compress_height" value="480"/>
</node>


</launch>
